Project/Core/Logger.cs

```diff
@@ -45,9 +40,27 @@ static Logger()
         /// </summary>
         public static Logger GetLogger => instance;
         // METHODS
-        private void Off(LogMode logMode)
+        /// <summary>
+        /// Turns off log modes.
+        /// <para/>
+        /// For multiple off modes use "|" between them, for example: <see cref="LogMode.Debug"/> | <see cref="LogMode.Info"/>.
+        /// </summary>
+        /// <param name="logMode">The log mode(s) that will be off.</param>
+        public void Off(LogMode logMode)
+        {
+            // Joins log modes that have to be off.
+            offLogMode |= logMode;
```

Comment:
look hard to understand, I would use Set instead, but if it works let it be, seems more efficient than Set

Reply:
it's optimization

Reply:
:ok_hand: